scripts/zones/Aht_Urhgan_Whitegate/npcs/Drahbah.lua

```diff
@@ -1,18 +1,1423 @@
 -----------------------------------
 -- Area: Aht Urhgan Whitegate
---  NPC: Drahbah
--- Standard Info NPC
+-- NPC: Drahbah
+-- Appraisal NPC
 -----------------------------------
+package.loaded["scripts/zones/Aht_Urhgan_Whitegate/IDs"] = nil
+local ID = require("scripts/zones/Aht_Urhgan_Whitegate/IDs")
+-----------------------------------
+require("scripts/globals/instance")
+
+local function appraiseItem(itemOrigin, itemType)
```

Comment:
This stuff should be tabled for ease of access from the NPC script. Store stuff in a way that makes sense, like by item id (instead of type), then origin, then the list of rewards. Can even store the chance for each reward.

Storing the data in this manner will remove this GIANT conditional chain and simplify the NPC's script.